moonraker/components/paneldue.py

```diff
@@ -38,12 +39,14 @@
 MIN_EST_TIME = 10.
 INITIALIZE_TIMEOUT = 10.
 
+
```

Comment:
Please don't submit changes to whitespace in pull requests.  Moonraker follows the same standard as Klipper here, code style and whitespace changes aren't accepted unless they are made by the original author of the module.